site/pages/docs/ref/bgimg/bgimg-en.hbs

```diff
@@ -0,0 +1,121 @@
+---
+{
+	"title": "Background image",
+	"language": "en",
+	"category": "Plugins",
+	"categoryfile": "plugins",
+	"description": "Apply a background image the URL given in paramenter or as per the srcset information.",
+	"altLangPrefix": "bgimg",
+	"dateModified": "2020-12-09"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Purpose</h2>
+<p>Apply a background image the URL given in paramenter or as per the srcset information.</p>
+
+<h2>Use when</h2>
+<ul>
+	<li>Display a background decorative image</li>
+</ul>
+
+<h2>Guidance for publisher</h2>
+<ul>
+	<li>Ensure the text contrast remain valid in the case where the image don't load or take time to load</li>
```

Comment:
Ensure the text contrast remains valid in the case where the image doesn't load or takes time to load